specs/AppNotifications/AppNotificationBuider/GroupAndSubgroup-spec.md

```diff
@@ -283,10 +294,17 @@ runtimeclass AppNotificationBuilder
+
+* Contrary to most other runtime classes part of the Builder API, the Subgroup runtime class doesn't expose its properties. This is in-line with the AppNotificationBuilder class which also does not expose properties and in part because there currently isn't a way to expose the Text strings along with their TextProperties that would make sense. If we created a struct to hold the text and their properties, we could easily expose them in the Subgroup class as well as in the AppNotificationBuilder class.
+
+* The Groups and Subgroups feature in AppNotifications offers users a wider range of text formatting options than those available in the main body of a notification. To enable support for these extended formatting options, we are proposing adding a new runtime class called AppNotificationExtendedTextProperties along eith one named AppNotificationExtendedImageProperties.
+
+  * We took care to give the new AppNotificationExtendedImageProperties a=nd AppNotificationExtendedTextProperties runtime classes names that is not specific to the Groups and Subgroups feature, ensuring that they can be easily adapted and re-used in the future as needed.
+
+  * While we initially considered having the new AppNotificationExtendedTextProperties class derive from the existing AppNotificationTextProperties class, doing so requires unsealing the later, which would break the contract. Also, there is a bug in the MIDL compiler that prevented us from proceeding with this approach. Nonetheless, where appropriate, the two classes will share code internally to ensure consistent behavior.
+
   * Although we also considered adding the new text formatting options directly to the existing AppNotificationTextProperties API, we ultimately decided that doing so could potentially create confusion, as some of the formatting options may not be applicable in all scenarios. As a result, we determined that the creation of a new runtime class specifically designed to handle the extended formatting options was the most efficient and effective solution.
+
+* Since we have to modify the contract on the AppNotificationBuilder, we are proposing to use this opportunity to better align the way text and image properties work. Currently, text properties are spercified in a runtime class while image properties are simply set when the image is added to the AppNotification. We are effectively proposing to deprecate the many variations of the image setting APIs on the AppNotificationBUilder and replacing them with a single call that takes an AppNotificationImageProperties object. This means that both: text and images work in the same way and ensure better future proofing.
```

Comment:
Line lengths are pretty long. Please [keep them to 100 chars](https://github.com/microsoft/WindowsAppSDK/blob/main/docs/Coding-Guidelines/Languages-Markdown.md#markdown)

VSCode has a nice extension that word wraps paragraphs smartly (don't split URLs etc)